README.md

```diff
@@ -436,6 +436,8 @@ test_project:
       location: "s3://aws-dbt-glue-datalake-1234567890-eu-west-1/"
       conf: spark.serializer=org.apache.spark.serializer.KryoSerializer --conf spark.sql.hive.convertMetastoreParquet=false
       datalake_formats: hudi
+      extra_jars: s3://{bucket}/configs/jars/hudi-utilities-bundle_2.12-0.12.1.jar,s3://{bucket}/configs/jars/hudi-spark3.1-bundle_2.12-0.12.1.jar,s3://{bucket}/configs/jars/spark-avro_2.12-3.2.2.jar,s3://{bucket}/configs/jars/calcite-core-1.32.0.jar
```

Comment:
Can we use some different example? Here are some reasons:
* Hudi's dependency is bit too complex as an example.
* Spark version corresponding to Glue 4.0 is 3.3.0.